### DISCUSSION TITLE
Ranking Leads on the website

### DISCUSSION THREAD

**Discussion by Utkarshbhimte** [covidarmy/backend#31] [Ideas]
We should look into making sure the leads on our website are valid. 

## Approaches
* Let the users update the status of the number, and we downrank those numbers temporarily

**Comment by rgxai** [comment]
Agreed. @arn4v can we have these 4 prompts? Will be putting this 4 prompts in chatbot too. I will create an issue for the same on FE.

---

Helpful: if they pick up and say that they have stock

Unresponsive: if the number is out of reach, busy or did not pick up

Out Of Stock: if they don’t have any more of the resource type available

 Invalid: if the number is wrong and they are not providing the selected resource type
